grape/general_graph.py

```diff
@@ -175,41 +151,22 @@ def construct_path_kernel(self, pred, nodi):
 
         paths = {}
 
-        for i in nodi:
+        for i in nodes:
             paths[self.ids[i]] = {
                 self.ids[j]: self.construct_path(i,j,pred)
```

Comment:
I'm a bit confused: why in this case the `..._kernel` method calls the generic method (`construct_path`) and not viceversa, as for others measures?

Reply:
Because of old naming and non-optimal encapsulation. Corrected. Now the calling "flux" is alike oher measures.